packages/animation-curve-block/tailwind.config.js

```diff
@@ -2,7 +2,11 @@
 
 module.exports = {
     presets: [require('@frontify/fondue/tailwind'), require('../shared/tailwind.config')],
-    content: ['src/**/*.{ts,tsx}', '../shared/src/**/*.{ts,tsx}'],
+    content: [
+        'src/**/*.{ts,tsx}',
+        '../shared/src/**/*.{ts,tsx}',
+        'node_modules/@frontify/guideline-blocks-settings/dist/**/*.es*.js',
```

Comment:
I think there should be a better solution than exporting from the `node_modules`

Reply:
We do the same in web-app. 🤔

Reply:
Ok maybe we should open a task to solve it on both ends, it's just really akward

Reply:
We need to import `import '@frontify/guideline-blocks-settings/styles';` so it won't be needed anymore